src/update_handler.c

```diff
@@ -1622,6 +1738,11 @@ RaucUpdatePair updatepairs[] = {
 	{"*.vfat", "boot-mbr-switch", img_to_boot_mbr_switch_handler},
 	{"*.img", "boot-mbr-switch", img_to_boot_mbr_switch_handler},
 	{"*", "boot-mbr-switch", NULL},
+#if ENABLE_GPT == 1
+	{"*.vfat", "boot-gpt-switch", img_to_boot_mbr_switch_handler},
```

Comment:
Here is a copy-n-paste leftover:
```suggestion
	{"*.vfat", "boot-gpt-switch", img_to_boot_gpt_switch_handler},
```